Source/WebKit/UIProcess/Launcher/haiku/ProcessLauncherHaiku.cpp

```diff
@@ -54,7 +68,9 @@ void ProcessLauncher::launchProcess()
         return;
     }
 
-	BString processIdentifier;
+	BString processIdentifier,processID;
+	team_id UIProcessID = getpid();
```

Comment:
Haiku way of getting team id exists something like be_app->teamid() i was too tired so didnt score through the docs 😄

Reply:
getpid is just fine.